package.json

```diff
@@ -59,12 +59,15 @@
     ]
   },
   "dependencies": {
+    "@types/bunyan": "^1.8.0",
```

Comment:
I think we should stick to the current stack: babel + Flow, and don't add any ts stuff. otherwise it will ends up  big porridge